example/lib/main.dart

```diff
@@ -130,6 +130,7 @@ class _MyHomePageState extends State<MyHomePage> {
               flex: 1,
             ),
             Text('msg').tr(args: ['aissat', 'Flutter']),
+            Text('msg_named').tr(namedArgs: {'lang': 'English'}),
```

Comment:
Text('msg_named').tr(namedArgs: {'lang': 'Dart'}),

Reply:
@3ace need to multimple localization

Reply:
@3ace do you mean the localization for `msg_named` in other language?

could you help with the other string localization? or should I use google translate for that purpose?

Reply:
> @3ace do you mean the localization for `msg_named` in other language?

Yes.
See comments to en-US.json